api/opds.py

```diff
@@ -786,6 +788,25 @@ def annotate_work_entry(
                 ),
             )
 
+        # Only OPDS for Distributors should get the time tracking link
+        if (
+            edition.medium == EditionConstants.AUDIO_MEDIUM
+            and active_license_pool
+            and active_license_pool.collection.protocol
+            == ExternalIntegration.OPDS_FOR_DISTRIBUTORS
```

Comment:
We want these entries only for books in the loan feed -- and the single entry feed we get after a borrow -- and only for loan (not hold) entries.

Reply:
Moved this to LibraryLoanAndHoldAnnotator